Dockerfile

```diff
@@ -0,0 +1,96 @@
+# parts of CAM require x86 architecture (gptl, which relies on the rdtsc x86 assembly instruction)
+# esmf is am image you are expected to have built. Read the README file for instructions
+FROM esmf:latest
+
+###################################################
+## Install necessary packages
+###################################################
+RUN dnf -y update \
+    && dnf -y install \
+      blas-devel \
+      cmake \
+      ftp \
+      gcc-c++ \
+      gcc-gfortran \
+      git \
+      hostname \
+      lapack-devel \ 
+      m4 \
+      make \
+      mpich \
+      mpich-devel \
+      netcdf-devel \
+      netcdf-fortran-devel \
+      python \
+      sudo \
+      svn \
+      tree \
+      vim \
+      wget \
+    && dnf clean all
```

Comment:
This is probably my Dockerfile ignorance, but don't many of these packages already exist in the `esmf` image?  If so, then do we need to re-install them here?

Reply:
No, you're correct. Some of them do exist. I initially had a single image for cam-sima and esmf and then I split them up. I think I didn't do a detailed job in installing on the required packages. I'll remove duplicates and ensure that it still works